.travis.yml

```diff
@@ -11,6 +11,8 @@ matrix:
         - os: linux
           python: 3.5
         - os: linux
+          python: 3.6
+        - os: linux
```

Comment:
This *appears* to be a change in the way the default CFLAGS from `_sysconfigdata` get used. I've verified that the typedefs for `tp_basicsize` haven't changed from 3.5 to 3.6 (at least on Mac). Both versions of `_sysconfigdata` (for me) specify `-Wsign-compare` which triggers the warning; that's only used, though, if I don't have CFLAGS set in my own environment (so I can trigger the warning on both 3.5 and 3.6). But when tox does the builds, I only get the warning for 3.6...so something has changed in the way the CFLAGS are set/used/cleared/something.

tl;dr: Not a new problem.